website/data/docs-navigation.js

```diff
@@ -7,183 +7,87 @@
-          },
-          {
-            category: 'binding-rule',
-            content: ['create', 'delete', 'list', 'read', 'update'],
-          },
-          'bootstrap',
-          {
-            category: 'policy',
-            content: ['create', 'delete', 'list', 'read', 'update'],
-          },
-          {
-            category: 'role',
-            content: ['create', 'delete', 'list', 'read', 'update'],
-          },
-          'set-agent-token',
-          {
-            category: 'token',
-            content: ['clone', 'create', 'delete', 'list', 'read', 'update'],
-          },
-          'translate-rules',
+          'zookeeper',
+          'chef-puppet',
+          'nagios-sensu',
+          'skydns',
+          'smartstack',
+          'serf',
+          'eureka',
+          'istio',
+          'proxies',
+          'custom'
         ],
       },
-      'agent',
-      { category: 'catalog', content: ['datacenters', 'nodes', 'services'] },
-      { category: 'config', content: ['delete', 'list', 'read', 'write'] },
-      { category: 'connect', content: ['ca', 'proxy', 'envoy', 'expose'] },
-      'debug',
-      'event',
-      'exec',
-      'force-leave',
-      'info',
-      {
-        category: 'intention',
-        content: ['check', 'create', 'delete', 'get', 'match'],
-      },
-      'join',
-      'keygen',
-      'keyring',
-      {
-        category: 'kv',
-        content: ['delete', 'export', 'get', 'import', 'put'],
-      },
-      'leave',
-      'license',
-      'lock',
-      'login',
-      'logout',
-      'maint',
-      'members',
-      'monitor',
-      {
-        category: 'namespace',
-        content: ['create', 'delete', 'list', 'read', 'update', 'write'],
-      },
-      {
-        category: 'operator',
-        content: ['area', 'autopilot', 'raft'],
-      },
-      'reload',
-      'rtt',
-      { category: 'services', content: ['register', 'deregister'] },
-      {
-        category: 'snapshot',
-        content: ['agent', 'inspect', 'restore', 'save'],
-      },
-      { category: 'tls', content: ['ca', 'cert'] },
-      'validate',
-      'version',
-      'watch',
-    ],
-  },
-  {
-    category: 'agent',
-    content: [
-      'dns',
-      'options',
-      {
-        category: 'config-entries',
-        content: [
-          'ingress-gateway',
-          'proxy-defaults',
-          'service-defaults',
-          'service-resolver',
-          'service-router',
-          'service-splitter',
-          'terminating-gateway',
-        ],
-      },
-      'cloud-auto-join',
-      'services',
-      'checks',
-      'kv',
-      'sentinel',
-      'encryption',
-      'telemetry',
-      'watches',
     ],
   },
+
+  { category: 'install', content: [
+    {title: 'Consul Agent', href: '/docs/agent'}, 
+  'concepts', 'ports', 'bootstrapping', 'cloud-auto-join', 'performance', {title: 'Kubernetes', href: '/docs/k8s'} ] },
```

Comment:
Should this sextion be collapsed in this way? It seems different from the rest.

Reply:
It's different because it has a few "outside" links that don't exist in the install category (Consul Agent and Kubernetes)

Reply:
Fine to resolve :)